oidc-service-impl/src/main/java/io/mosip/esignet/services/LinkedAuthorizationServiceImpl.java

```diff
@@ -211,6 +224,7 @@ public LinkedConsentResponse saveConsent(LinkedConsentRequest linkedConsentReque
 
         LinkedConsentResponse authRespDto = new LinkedConsentResponse();
         authRespDto.setLinkedTransactionId(linkedConsentRequest.getLinkedTransactionId());
+        auditWrapper.logAudit(Action.SAVE_CONSENT, ActionStatus.SUCCESS, AuditHelper.buildAuditDto(null, transaction), null);
```

Comment:
why is transactionId passed as null in AuditHelper.buildAuditDto?

Reply:
Passed linkedTransactionId as first parameter.